src/coreanimation.cs

```diff
@@ -50,6 +50,12 @@
 using Metal;
 using SceneKit; // For SCNAnimationEvent
 
+#if !MONOMAC
```

Comment:
Curious why these were added

Reply:
Check the types before removed out of #if blocks. These types are mac only, and thus they would not compile without stand ins.